packages/app/src/components/Survey/SurveyPresentational.tsx

```diff
@@ -99,15 +100,49 @@ export const SurveyPresentational: React.FC<SurveyPresentationalProps> = ({
 
                 <Box>
                     <Heading use="h3" marginBottom="major-3">
-                        {intl.formatMessage({
-                            id: "statistics",
-                            defaultMessage: "Statistics"
-                        })}
+                        Statistics
```

Comment:
Statistics is hardcoded. it must use the intl format. 

Please put statistics in the intl format like this 
```js 
{intl.formatMessage({
                            id: "statistics",
                            defaultMessage: "Statistics"
                        })}

                        ```